WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -0,0 +1,222 @@
+                }
+                .padding([.leading, .trailing], Layout.contentSpacing)
+                .padding([.top, .bottom], Layout.contentVerticalSpacing)
+                .background(Color(.systemGray6))
+
+                Spacer()
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .background(Color(.systemGray6))
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationTitle(Localization.adDestination)
+            .toolbar {
+                ToolbarItem(placement: .cancellationAction) {
+                    Button(Localization.cancel) {
+                        dismiss()
+                    }
+                }
+            }
+        }
+    }
+    private func sectionHeading(title: String) -> some View {
+        Text(title.uppercased())
+            .subheadlineStyle()
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding(Layout.sectionHeadingPadding)
+    }
+
+    private func destinationItem(title: String,
+                                 subtitle: String,
+                                 type: DestinationType,
+                                 showBottomDivider: Bool = false) -> some View {
+        HStack(alignment: .center) {
+            if type == viewModel.selectedDestinationType {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .foregroundColor(Color(uiColor: .accent))
+            } else {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .hidden() // Small hack to make the icon space consistent while not showing the icon.
+            }
+
+            VStack(alignment: .leading) {
+                Text(title)
+                    .bodyStyle()
+                    .padding(.top, Layout.contentVerticalSpacing)
+                Text(subtitle)
+                    .subheadlineStyle()
+                    .multilineTextAlignment(.leading)
+                    .padding(.bottom, Layout.contentVerticalSpacing)
+
+                if showBottomDivider {
+                    Divider()
+                        .background(Color(.systemGray3))
+                }
+            }
+        }
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .background(Color(.systemBackground))
+        .onTapGesture {
+            viewModel.setDestinationType(type: type)
+        }
+    }
+
+    private func parameterItem(itemName: String) -> some View {
+        VStack {
+            HStack {
+                Text(itemName)
+                    .padding([.top, .bottom], Layout.parametersVerticalSpacing)
+                Spacer()
+                Image(systemName: "chevron.right")
+                    .foregroundColor(Color(.systemGray4))
```

Comment:
We can use `secondary` for this color.

Reply:
I updated it in the next PR https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/b3f6fb97805bceead1ad25daafbcde3396a69621

secondary looks a bit too dark on my monitor compared to the design but it could just be my monitor.